[2022-03-15 08:03:18,711] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: glue_poc.create_bucket scheduled__2022-03-13T00:00:00+00:00 [queued]>
[2022-03-15 08:03:18,841] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: glue_poc.create_bucket scheduled__2022-03-13T00:00:00+00:00 [queued]>
[2022-03-15 08:03:18,844] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 08:03:18,850] {taskinstance.py:1244} INFO - Starting attempt 16 of 16
[2022-03-15 08:03:18,853] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 08:03:18,979] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): create_bucket> on 2022-03-13 00:00:00+00:00
[2022-03-15 08:03:19,011] {standard_task_runner.py:52} INFO - Started process 12791 to run task
[2022-03-15 08:03:19,081] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'glue_poc', 'create_bucket', 'scheduled__2022-03-13T00:00:00+00:00', '--job-id', '160', '--raw', '--subdir', 'DAGS_FOLDER/glue_poc.py', '--cfg-path', '/tmp/tmpoiweijzj', '--error-file', '/tmp/tmp95d9_5m6']
[2022-03-15 08:03:19,106] {standard_task_runner.py:77} INFO - Job 160: Subtask create_bucket
[2022-03-15 08:03:19,463] {logging_mixin.py:109} INFO - Running <TaskInstance: glue_poc.create_bucket scheduled__2022-03-13T00:00:00+00:00 [running]> on host cdd2ec182991
[2022-03-15 08:03:20,294] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=glue_poc
AIRFLOW_CTX_TASK_ID=create_bucket
AIRFLOW_CTX_EXECUTION_DATE=2022-03-13T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-13T00:00:00+00:00
[2022-03-15 08:03:20,457] {base_aws.py:194} INFO - Credentials retrieved from extra_config
[2022-03-15 08:03:26,817] {python.py:175} INFO - Done. Returned value was: None
[2022-03-15 08:03:27,138] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=glue_poc, task_id=create_bucket, execution_date=20220313T000000, start_date=20220315T080318, end_date=20220315T080327
[2022-03-15 08:03:28,159] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-15 08:03:29,207] {local_task_job.py:264} INFO - 2 downstream tasks scheduled from follow-on schedule check
